private/data/i18n/en.json

```diff
@@ -66,57 +180,83 @@
+        "createTag": "Create tag"
       },
       "productDetailEdit": {
+        "editOption": "Edit option",
+        "productSettings": "Product settings",
+        "product": "Product",
+        "size": "Size",
         "title": "Title",
+        "untitledProduct": "Untitled product",
+        "parmalink": "Permalink",
         "pageTitle": "Subtitle",
         "vendor": "Vendor",
         "description": "Description",
+        "originCountry": "Origin country",
         "facebookMsg": "Facebook message",
         "twitterMsg": "Twitter message",
         "pinterestMsg": "Pinterest message",
         "googleplusMsg": "Google+ message",
+        "editVariant": "Edit variant",
+        "variantOptions": "Variant options",
+        "variantDetails": "Variant details",
         "duplicateVariant": "Duplicate",
         "addVariantOption": "Add option",
-        "removeVariant": "Remove",
-        "duplicateProduct": "Duplicate product",
-        "pinProduct": "Pin product",
-        "toggleSize": "Toggle size",
-        "publish": "Publish",
-        "moreOptions": "More options",
-        "details": "Details",
         "addVariantFail": "Unable to create variant for {{title}}.",
         "cloneVariantFail": "Unable to clone variant for {{title}}.",
+        "removeVariant": "Remove",
         "removeVariantConfirm": "Are you sure you want to delete {{title}}",
-        "restoreVariantConfirm": "Are you sure you want to restore {{title}} from the vieve?",
+        "restoreVariantConfirm": "Are you sure you want to restore {{title}} from the archive?",
         "restoreVariantFail": "Unable to restore {{title}}.",
         "archiveVariantConfirm": "Are you sure you want to archive {{title}}",
         "thisVariant": "this variant",
         "thisOption": "this option",
+        "duplicateProduct": "Duplicate product",
+        "pinProduct": "Pin product",
+        "toggleSize": "Toggle Size",
         "archiveProduct": "Archive product",
         "restoreProduct": "Restore product from archive",
         "archiveThisProduct": "Archive this product?",
         "archiveSelectedProducts": "Archive selected products?",
+        "publish": "Publish",
+        "moreOptions": "More Options",
+        "details": "Details",
         "noSelectedProducts": "No selected products. Use the ",
         "orShiftCtrlCmd": " or shift|ctrl|cmd + click on products to select multiple. With shift you can select a contiguous range of products.",
         "makeVisible": "Make visible",
         "hideProduct": "Hide product",
         "productIsVisible": "Product is visible",
-        "productIsNotVisible": "Product is not visible",
-        "variantOptions": "Variant options"
+        "productIsNotVisible": "Product is not visible"
       },
       "productVariant": {
         "addVariantOptions": "Add options to enable 'Add to Cart' button",
         "inventoryQuantity": "Quantity",
-        "title": "Title",
+        "title": "Label",
```

Comment:
I remember this change, I think we had decided to keep "Title" through out the variants and options.